charts/invenio/templates/invenio-configmap.yaml

```diff
@@ -11,7 +11,10 @@ data:
   INVENIO_CELERY_RESULT_BACKEND: 'redis://{{ include "invenio.redis.hostname" . }}:6379/2'
   INVENIO_RATELIMIT_STORAGE_URL: 'redis://{{ include "invenio.redis.hostname" . }}:6379/3'
   INVENIO_COMMUNITIES_IDENTITIES_CACHE_REDIS_URL: 'redis://{{ include "invenio.redis.hostname" . }}:6379/4'
+  {{- if hasKey .Values.invenio.extra_config "INVENIO_SEARCH_HOSTS" }}
+  {{- else }}
```

Comment:
Instead of having an empty if statement, I think it is preferable to negate the condition using [`not`](https://helm.sh/docs/chart_template_guide/function_list/#not).